tests/test_features/test_projection.py

```diff
@@ -0,0 +1,209 @@
+import pytest
+import matplotlib.pyplot as plt
+
+from yellowbrick.features.projection import ProjectionVisualizer
+from yellowbrick.exceptions import YellowbrickValueError
+
+from tests.base import VisualTestCase
+from ..fixtures import Dataset
+from unittest import mock
+
+from sklearn.datasets import make_classification, make_regression
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import  StandardScaler
+from sklearn.decomposition import PCA
+from sklearn.manifold import LocallyLinearEmbedding
+
+##########################################################################
+## Fixtures
+##########################################################################
+
+@pytest.fixture(scope="class")
+def discrete(request):
+    """
+    Creare a random classification fixture.
+    """
+    X, y = make_classification(
+        n_samples=400, n_features=12, n_informative=10, n_redundant=0,
+        n_classes=5, random_state=2019)
+
+    # Set a class attribute for discrete data
+    request.cls.discrete = Dataset(X, y)
+
+@pytest.fixture(scope="class")
+def continuous(request):
+    """
+    Creates a random regressor fixture.
+    """
+    X, y = make_regression(
+        n_samples=500, n_features=22, n_informative=8, random_state=2019
+    )
+
+    # Set a class attribute for continuous data
+    request.cls.continuous = Dataset(X, y)
+
+@pytest.fixture(scope="function")
+def transformer(request):
+    """
+    Creates a PCA transformer. Acccepts the projection dimnesion as param and
```

Comment:
Sorry for the inconvenience. I will keep it in mind.
This function returns `none()` for a specific reason. This fixture has a scope 'function'. And I used it for the class. Which means that it is called for every function of the class. Some of the tests do not need this fixture and which is ultimately called and lead to error. Hence to handle this error I returned  `None`. 
But by your above suggestion, this will ultimately be removed.